docs/repo/property-pages/visibility-conditions.md

```diff
@@ -18,6 +23,12 @@ In a XAML rule file, a visibility condition is specified as metadata on the prop
     <NameValuePair Name="VisibilityCondition">
       <NameValuePair.Value>(has-evaluated-value "MyPage" "MyProperty" "Foo")</NameValuePair.Value>
     </NameValuePair>
+    <NameValuePair Name="AllowedVaryByDimensions">
+        <NameValuePair.Value>(matches "Configuration" this)</NameValuePair.Value> <!-- 'this' refers to each dimension candidate -->
```

Comment:
Following comments on the (internal) CPS PR, this can be written as:

```suggestion
        <NameValuePair.Value>(eq (dimension) "Configuration")</NameValuePair.Value>
```